packages/dd-trace/src/config.js

```diff
@@ -522,6 +522,18 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
       true
     )
 
+    const DD_EXPERIMENTAL_API_SECURITY_ENABLED = coalesce(
+      options.appsec?.apiSecurity?.enabled,
+      process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED && isTrue(process.env.DD_EXPERIMENTAL_API_SECURITY_ENABLED),
+      false
```

Comment:
`isTrue` expects a variable but does not check whether is undefined or not so I think if `DD_EXPERIMENTAL_API_SECURITY_ENABLED` is not defined it will throw

Reply:
Try it, I guarantee you it will work, and I don't even need to try it to say it

Reply:
oh, interesting an undefined value got casted to 'undefined'